keras/layers/core.py

```diff
@@ -100,6 +100,8 @@ class Dropout(Layer):
     @interfaces.legacy_dropout_support
     def __init__(self, rate, noise_shape=None, seed=None, **kwargs):
         super(Dropout, self).__init__(**kwargs)
+        if (rate > 1) or (rate < 0):
+            warnings.warn('Dropout `rate` should be between 0 and 1')
```

Comment:
This is something that should be mentioned in the documentation (the docstring) rather than displayed in a warning. Is it not in the docs already?

Reply:
Each dropout has its own docstring that explains it. It just seemed like there is never a reason it should be greater than 1 and beginners might benefit from a warning message.